### PR TITLE
Add SHIELD_ADVANCED_ENABLED_AUTORENEW

### PR DESCRIPTION
**Pull request by schirag1993**
I confirm these files are made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)

*Issue #, if available:*
#198 

*Description of changes:*
N/A

### CODE REVIEW

File: python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+# the specific language governing permissions and limitations under the License.
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  SHIELD_ADVANCED_ENABLED_AUTORENEW
+
+Description:
+  Check whether AWS Shield Advanced is enabled and is set to autorenew. This service's API is only available in us-east-1
```
[the remark this answers is not in the document]
Comment (schirag1993) [reply python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py]: Done!

File: python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+
+def evaluate_parameters(rule_parameters):
+    """Evaluate the rule parameters dictionary validity. Raise a ValueError for invalid parameters.
+
+    Return:
+    anything suitable for the evaluate_compliance()
+
+    Keyword arguments:
+    rule_parameters -- the Key/Value dictionary of the Config Rules parameters
+    """
```
[the remark this answers is not in the document]
Comment (schirag1993) [reply python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py]: Done!

**Comment by santosh07bec** [comment]
Looks good to me now.
@jongogogo, please check the code and let us know if you have any suggestions.

File: python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+def shield_advanced_check(client):
+    try:
+        return client.describe_subscription()['Subscription']['AutoRenew']
+    except botocore.exceptions.ClientError as error:
+        return error.response['Error']['Code']
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    shield_advanced_enabled = shield_advanced_check(get_client('shield', event))
+    if shield_advanced_enabled == 'ResourceNotFoundException':
+        return [build_evaluation(event['accountId'],
```
Comment (jongogogo) [inline python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py]: No need to return a list. Idem for the rest

Comment (schirag1993) [reply python/SHIELD_ADVANCED_ENABLED_AUTORENEW/SHIELD_ADVANCED_ENABLED_AUTORENEW.py]: @jongogogo, it's done. Kindly review it

**Pull request merged**